var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -156,6 +156,13 @@ def libs(self):
             libraries, root=self.prefix, shared=shared, recursive=True
         )
 
+    variant('context-impl',
+            default='fcontext',
+            values=('fcontext','ucontext','winfib'),
```

Comment:
Please correct style issues:

```suggestion
            values=('fcontext', 'ucontext', 'winfib'),
```

Reply:
Sorted in [65d7b34](https://github.com/spack/spack/pull/30496/commits/65d7b34ce1839e576f377bba64a5eb9c256a2a58)